### DISCUSSION TITLE
Crazyradio

### DISCUSSION THREAD

**Discussion by CalvinKoung** [bitcraze/discussions#1246] [Q&A]
Hi, I can saw the crazyradio 2.0 in my USB port but I start the python program the terminal show the "cannot find a crazyradio dongle". How can I fix it ? thanks.

**Comment by knmcguire** [comment]
Hi! is this a completely new Crazyradio 2.0 ? Have you followed [this tutorial](https://www.bitcraze.io/documentation/tutorials/getting-started-with-crazyradio-2-0/) for it?

**Reply by CalvinKoung** [reply]
Yes, it will appear ERROR:cflib.crazyflie:Couldn't load link driver: Cannot find a Crazyradio Dongle. But my laptop can found it. 

![螢幕擷取畫面 2024-03-25 184044](https://github.com/bitcraze/discussions/assets/118436188/d2ec9330-2bbb-46b0-9c87-a8664f1c46a4)

**Reply by knmcguire** [reply]
So you are using windows. Have you followed the very last step of the tutorial? There are some OS specific things you need to install/configure as well: https://www.bitcraze.io/documentation/tutorials/getting-started-with-crazyradio-2-0/#install-and-configure

**Reply by CalvinKoung** [reply]
Ok, I thought I install the wrong driver.

**Reply by knmcguire** [reply]
it could be but it is important that you still follow the Zadig driver instructions since the crazyradio 2.0 is in 'Crazyradio 1.0 emulator mode' 

Keep us in the loop if this doesn't work out
